src/features/ISECompatibility.ts

```diff
@@ -24,6 +24,7 @@ export class ISECompatibilityFeature implements IFeature {
         { path: "powershell.integratedConsole", name: "focusConsoleOnExecute", value: false },
         { path: "files", name: "defaultLanguage", value: "powershell" },
         { path: "workbench", name: "colorTheme", value: "PowerShell ISE" },
+        { path: "editor", name: "wordSeparators", value: "`~!@#%^&*()-=+[{]}\\|;:'\",.<>/?" }
```

Comment:
Did this value for the word separators come from anywhere in particular? I'm wondering about the following: `~`, `@`, `#`, `^`, `*`, `-`, `\`, `:`.

It depends on the context in which I think of where we're defining "words" too. In expression mode, `*` and `-` are word separators, but in command mode they're not. I suspect we should prefer expression mode, but want to raise it in any case

Reply:
It's the default minus the `$` - I originally said this here:
https://github.com/PowerShell/vscode-powershell/issues/2546#issuecomment-597328583

Not perfect... but better.

Reply:
Right, i used the example Tyler mentioned in the comment.
Is there a better implementation?

Reply:
I'm happy with better. We can improve it in future as needed